.gitignore

```
@@ -124,4 +124,9 @@ squad*
 
 # nodejs specific
 node_modules
-sockets
\ No newline at end of file
+sockets
+
+
+.env.development
```

.env가 여러 형태로 존재할 수 있으므로 glob 패턴을 써서 표준화해야 합니다.
```suggestion
.env.*
```

---

.env.*를 명시하게 되면, sample env파일로 필터링되지 않나요?

---

맞습니다. 각각 적히거나, glob 패턴을 활용해서 `.env.sample`만 허용하여야 합니다.

---

각각 적겠습니다!